packages/atomic/src/components/result-lists/result-list-common.pcss

```diff
@@ -205,10 +205,10 @@
     }
 
     &.image-small {
-      @media not all and (min-width: 1180px) {
+      @media not all and (min-width: 1024px) {
         grid-template-columns: repeat(3, 1fr);
       }
-      @media (min-width: 1180px) {
+      @media (min-width: 1024px) {
```

Comment:
The default mobile breakpoint in `atomic-search-layout` is 1024px. Setting `1024px` in this file means that it becomes customizable. I don't think that's a problem though? Mentioning it in case anybody else thinks that may be a problem.